webui/react/src/routes/routes.ts

```diff
@@ -154,12 +154,12 @@ const routes: RouteConfig[] = [
   {
     id: 'resourcepool',
     needAuth: true,
-    path: '/resourcepool/:poolname',
+    path: '/resourcepool/:poolname/:tab',
   },
   {
     id: 'resourcepool',
     needAuth: true,
-    path: '/resourcepool/:poolname/:tab',
+    path: '/resourcepool/:poolname',
```

Comment:
React-router will use the first matching route in the array, not the best match. This means that if the route without `/:tab` is first, it will always be the route matched. That leads to not being able to get the value of `tab` from `useParams`